.github/workflows/build_and_test.yml

```diff
@@ -42,3 +42,32 @@ jobs:
             --packages-up-to obstacle_stop_planner_refine \
             --event-handlers console_cohesion+ \
             --return-code-on-test-failure
+
+  code-metrics:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+
+      - name: Measure code metrics
+        run: |
+          git clone https://github.com/terryyin/lizard.git
+          mkdir public
+          python3 lizard/lizard.py obstacle_stop_planner_refine/ -l cpp -x"*test*" \
+            --CCN 20 --length 200 --arguments 6 \
+            --html > public/lizard_only_wo_test.html || true
+          python3 lizard/lizard.py obstacle_stop_planner_refine/ -l cpp -x"*test*" \
+            --CCN 20 --length 200 --arguments 6 || true
+
+      - name: Upload metrics
```

Comment:
IMHO:
- `Upload measurement result page`
- `Deploy measurement result page to gh-pages`
would be easier to understand each task.

Reply:
Changed 7901092 :+1: